src/gradualizer_db.erl

```diff
@@ -459,10 +472,30 @@ get_src_map() ->
                    RevPath         -> lists:reverse("lre.*/" ++ RevPath)
                end || Path <- code:get_path()],
     SrcFiles = lists:flatmap(fun filelib:wildcard/1, SrcDirs),
-    {ok, RE} = re:compile(<<"([^/.]*)\.erl$">>),
     Pairs = [begin
-                 {match, [Mod]} = re:run(Filename, RE,
+                 {match, [Mod]} = re:run(Filename, erl_file_regexp(),
```

Comment:
Are there regexp sigils in Erlang? For example in Elixir you can just write `~r/^hello$/` and it will be compiled in compile-time and inlined to code as value